### ISSUE TITLE
Do not show meta data when printing TIR

### ISSUE THREAD

**Issue by merrymercy** [apache/incubator-tvm#6878]
Recently I found TVM will print meta when I do 
```python
print(tvm.lower(s, [A, B, C], simple_mode=True))
```

Here is an example output in the tutorial website (https://tvm.apache.org/docs/tutorials/language/schedule_primitives.html)
```
#[version = "0.0.5"]
primfn(A_1: handle, B_1: handle, C_1: handle) -> ()
  attr = {"global_symbol": "main", "tir.noalias": True}
  buffers = {C: Buffer(C_2: Pointer(float32), float32, [m: int32, n: int32], [stride: int32, stride_1: int32], type="auto"),
             B: Buffer(B_2: Pointer(float32), float32, [m, n], [stride_2: int32, stride_3: int32], type="auto"),
             A: Buffer(A_2: Pointer(float32), float32, [m, n], [stride_4: int32, stride_5: int32], type="auto")}
  buffer_map = {A_1: A, B_1: B, C_1: C} {
  for (i: int32, 0, m) {
    for (j: int32, 0, n) {
      C_2[((i*stride) + (j*stride_1))] = ((float32*)A_2[((i*stride_4) + (j*stride_5))]*(float32*)B_2[((i*stride_2) + (j*stride_3))])
    }
  }
}

#[metadata]
{
  "root": 1,
  "nodes": [
    {
      "type_key": ""
    },
    {
      "type_key": "Map",
      "keys": [
        "IntImm"
      ],
      "data": [2]
    },
    {
      "type_key": "Array",
      "data": [3]
    },
    {
      "type_key": "IntImm",
      "attrs": {
        "dtype": "bool",
        "value": "1"
      }
    }
  ],
  "b64ndarrays": [],
  "attrs": {"tvm_version": "0.8.dev0"}
}
```

I found this very annoying.  Why do we print it? Can we disable it for this use case by default?
cc @tqchen @spectrometerHBH @jroesch @junrushao1994

**Comment by junrushao1994** [comment]
You may slightly these lines: https://github.com/apache/incubator-tvm/blob/main/python/tvm/ir/module.py#L253-L258, adding `show_meta_data=False` to `self.astext`

**Comment by junrushao1994** [comment]
@jroesch shall we turn off the `show_meta_data` in `IRModule.__str__` and `IRModule.__repr__`? If so, @merrymercy would you mind send a PR?

**Comment by tqchen** [comment]
I agree that for repr and str we should turn off the meta data since they are not strictly required

**Comment by merrymercy** [comment]
Fixed by #6881
